README.md

```diff
@@ -38,15 +38,17 @@ This repository is home to the `sgdemux` tool for demultiplexing sequencing data
 
 ### From Bioconda
 
-Install from [`bioconda`](https://bioconda.github.io/) with:
+Install from [`bioconda`](https://bioconda.github.io/) with following command. Please create a new Conda environment for sgdemux to prevent dependency clashes. 
 
 ```console
 conda install -c bioconda sgdemux
```

Comment:
If we're suggesting to create a new conda environment, then I'd recommend that we describe the steps instead
```suggestion
Install from [`bioconda`](https://bioconda.github.io/) with:

```console
conda create -n sgdemux -c bioconda sgdemux
conda activate sgdemux
```